ui/app/AppLayouts/Onboarding2/OnboardingLayout.qml

```diff
@@ -44,10 +47,11 @@ Page {
     function restartFlow() {
         unload()
 
-        if (loginAccountsModel.ModelCount.empty)
-            onboardingFlow.init()
-        else
-            stack.push(loginScreenComponent)
+        onboardingFlow.init()
+        // if (loginAccountsModel.ModelCount.empty)
```

Comment:
Leftovers?

Reply:
yeah indeed, it shouldn't be here; probably a bad rebase

Reply:
Yeah, sorry I'll rebase in a few

Reply:
Done